tensorflow-sys/tests/lib.rs

```diff
@@ -8,3 +9,32 @@ fn linkage() {
         ffi::TF_DeleteBuffer(buffer);
     }
 }
+
+/// Test that the eager API works.
+#[cfg(feature = "eager")]
+#[test]
+fn tfe_tensor_handle() {
+    let data = vec![0.0f32; 100];
+    let shape = [1, 10, 10];
+    let num_elements = shape.iter().fold(1, |t, x| t * x) as usize;
```

Comment:
This can be simplified to `shape.iter().product()`.

Reply:
Thanks. The code now uses `product`.